src/AstAnalyser.js

```diff
@@ -1,6 +1,7 @@
 // Import Node.js Dependencies
 import fs from "node:fs/promises";
 import path from "node:path";
+import assert from "node:assert";
```

Comment:
Remove that line pls.
Don't forget to run `npm run lint` if you want to make sure that the code is correctly formatted before push.